apps/src/blockly/addons/cdoTheme.js

```diff
@@ -1,21 +1,53 @@
 import GoogleBlockly from 'blockly/core';
 
+// Intentionally override definition from core Blockly
```

Comment:
Just out of curiosity, where do these overrides come from? Is this just to match current site behavior?